src/spike_source_cell_group.cpp

```diff
@@ -0,0 +1,72 @@
+#include <exception>
+
+#include <cell_group.hpp>
+#include <profiling/profiler.hpp>
+#include <recipe.hpp>
+#include <spike_source_cell_group.hpp>
+#include <time_sequence.hpp>
+
+namespace arb {
+
+spike_source_cell_group::spike_source_cell_group(std::vector<cell_gid_type> gids, const recipe& rec):
+    gids_(std::move(gids))
+{
+    time_sequences_.reserve(gids_.size());
+    for (auto gid: gids_) {
+        time_sequences_.push_back(util::any_cast<time_seq>(rec.get_cell_description(gid)));
```

Comment:
Because a `time_seq` is more general in application than a spike source 'cell', and because they are essentially an implementation detail for the description of a spike source, I think they should have a (trivial) wrapper, e.g. something like
```
struct spike_source {
    explicit spike_source(time_seq times): times_(std::move(times)) {}
    time_seq times_;
};
```
It keeps the recipe usage consistent — each cell group kind has its own description class — and helps the code document itself.

Reply:
Done. There is now a `spike_source_cell` wrapper kind.